src/applications/vaos/services/patient/index.js

```diff
@@ -156,7 +157,44 @@ export async function fetchPatientEligibility({
   typeOfCare,
   location,
   type = null,
+  useV2 = false,
 }) {
+  if (useV2) {
```

Comment:
I code put this in its own function like `fetchPatientEligibilityFromVAR`, but it'll eventually replace `fetchPatientEligibilityFromVAR` and then a separate function won't really make sense.